src/alcotest/QCheck_alcotest.mli

```diff
@@ -12,7 +12,10 @@
 *)
 
 val to_alcotest :
-  ?verbose:bool -> ?long:bool -> ?rand:Random.State.t ->
+  ?colors:bool -> ?verbose:bool -> ?long:bool ->
+  ?debug_shrink:(out_channel option) ->
+  ?debug_shrink_list:(string list) ->
+  ?rand:Random.State.t ->
   QCheck2.Test.t -> unit Alcotest.test_case
```

Comment:
Even though already merged:
Could I ask you to document the new params in the .mli @gasche - pretty please? 😀🙏‍

Reply:
Sure! But then again, those are the same parameters as [QCheck_base_runner.run_tests](http://c-cube.github.io/qcheck/0.18/qcheck-core/QCheck_base_runner/index.html#val-run_tests). (I can just add a cross-reference in the documentation.)

Reply:
A cross-reference is fine! 
I wrote that documentation for `run_tests` only a few months back since it was missing, hence my knee-jerk missing-doc reaction 😅 I missed that empty-list was special cased though... 😬

Reply:
Done in #221